01-main/packages/popcorn-time

```diff
@@ -0,0 +1,9 @@
+DEFVER=1
+get_github_releases "popcorn-official/popcorn-desktop" "latest"
+if [ "${ACTION}" != "prettylist" ]; then
+    URL="$(grep "browser_download_url.*\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d'"' -f4)"
```

Comment:
better not to just grab the first deb from a selection - if none specific to OS or release but multiple ARCHs are available then I'd suggest to filter by HOST_ARCH